controller/Dockerfile

```diff
@@ -128,7 +133,7 @@ ENTRYPOINT ["./retina/controller"]
 
 
 # agent final image for windows 
-FROM ${OS_VERSION} AS agent-win
+FROM ${OS_VERSION} as agent-win
```

Comment:
```suggestion
FROM ${OS_VERSION} AS agent-win
```

Reply:
ARGH… I stared at those lines for like 20 minutes trying to figure out where the conflict was and it was capitalization *in* the line 🤦